index.bs

```diff
@@ -1457,6 +1459,79 @@ Resignation from a Group</h3>
 	See “Exclusion and Resignation from the Working Group” in the W3C Patent Policy [[PATENT-POLICY]]
 	for information about obligations remaining after resignation from certain groups.
 
+<h3 id="tooling">
+Tooling</h3>
+
+	For [=W3C Groups=] operating under this Process,
+	a core operating principle is to allow access across disabilities,
+	across country borders,
+	and across time.
+	Thus in order to allow all would-be participants to effectively participate,
+	to allow future participants and observers to understand the rationale and origins of current decisions,
+	and to guarantee long-lived access to its publications,
+	W3C requires that:
+
+	<ul>
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			(i.e. intended for use or reference outside its own membership)
+			<em class=rfc2119>must</em> be published and promoted at a W3C-controlled URL,
+			and backed up by W3C systems
+			such that if the underlying service is discontinued,
+			W3C can continue to serve such content without breaking incoming links
+			or other key functionality.
+
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			<em class=rfc2119>must</em> follow best practices for accessibility worldwide
+			and for persons with disabilities.
+			Network access to w3.org itself may be assumed.
+
+		<li>
+			Persistent text-based discussions sponsored by the group
+			pertaining to their work
+			and intended to be referenceable by all group members,
+			including official meeting minutes and especially records of decisions made,
+			<em class=rfc2119>must</em> be archived by W3C for future reference.
+			This includes discussions conducted over email lists
+			or in issue-tracking services
+			or any equivalent fora.
+			Any discussion system whose archive is separate from its active interactive operation
+			<em class=rfc2119>must</em> make that archive available
+			at the same level of confidentiality.
+
+		<li>
+			Any tooling used by the group
+			for producing its documentation and deliverables
+			or for official group discussions
+			<em class=rfc2119>must</em> be usable
+			without cost (direct or indirect)
+			by all active or would-be active group participants
+			to allow their effective participation
+			regardless of disability
+			or geographical location.
```

Comment:
I think a really hard question here is not so much about geographical location as much as administrative prohibitions of usage, many linked to governmental decisions (loosely bound to geographical locations), but some linked to very different authorities (e.g. organizational ban on certain tools).

I think the rule in the process needs to address that aspect more than geography (which as far as I know is rarely an issue); and the associated separate guidance should probably hints to how far a group needs to adapt the many edge cases that arise when applying the rule.

Reply:
I think we mainly care about geographical restrictions. I don't think we can mandate accommodating all of the IT teams of the world (though we can certainly take that into account voluntarily). But I think we should take into consideration the governmental bans, because there's no way practical around that.

Reply:
And I think [we must not be held hostage to government bans](https://github.com/w3c/w3process/issues/435#issuecomment-672267652), lest [they become a means for a downgrade attack.](https://github.com/w3c/w3process/issues/435#issuecomment-717924343).

Reply:
Not being held hostage to government bans is a nice idea. But for an organisation that works primarily with organisations who are subject to various local government rules, it makes no sense in practice.

We should pay attention to the concerns of actual people who are unable to use tools because their employer (or similar) will not allow it. The workarounds I have dealt with often involve having to set up a separate machine (which has a direct cost), an alternative email system which works poorly with the general W3C assumption that we can identify who represents what member, and other inconveniences that add up to a notable barrier to participation. For someone who is concerned their issues are likely to be very hard to move forward anyway, this is a pretty significant disincentive to participation.

Reply:
I agree with @chaals' first paragraph in the comment above.

As for the situation discussed in the second one, I think this is one where being accomodating is good, but I don't think we want to constrain it with RFC2119 language in the Process. It probably does make for a relevant discussion in /Guide though.

Reply:
I agree with Sam for a couple of reasons. First if your tool choice is determined by how authoritarian censors behave then you're at the whim of dictatorial decision making. Second you might be inadvertently trading off better access for some over others. There will be those in places where blocking and filtering uses one tactic (TLS 1.3 blocking, for example), and those in places where blocking and filtering uses another tactic that the same protocol makes the tool resistant to censorship (DNS/domain blocking, respectively in this example).

Suggest thinking of proactive anti-censorship mechanisms that provide blanket privacy, and therefore censorship resistance, to the most people, like tools that provide an onion service, or actually the W3C participating in the Tor network through bridges and relays (if it doesn't already).

Reply:
It's phrased as a 'should' so that we (a) can think about and examine hard cases and (b) decide, when appropriate, not to follow it. So there may be cases where we regretfully decide that some members will have problems, but it's the right choice. But that should be a conscious decision; we do not want the case where some members can't access, and no-one cares or has thought about it.